File: Arrays/1.Arrays and array_list/Arrays.py
arrays=[1,2,3,4,5]
arrays1=[1,2,'Hello','Everyone',5]
print(arrays)
print(arrays1)
# Accessing th elements
print(arrays[1])
# Adding the elements
arrays.append(40)
#Poping the element with the index value
arrays.pop(4)
#Removing the element 
arrays.remove(4)
#Deleting the  element with the index value
del arrays[0]
#inseting the index value
arrays.insert(0,"Hello")
#Extending the values in list
arrays.extend([20,30,40])
print(arrays)

File: 1.Arrays/1.Arrays and array_list/Arrays.java
public class Arrays {
    public static void main(String[] args) {
        int[] arrays={10,20,30};
        // Traverse the arrays
        for (int i=0;i<arrays.length;i++){
            System.out.println(arrays[i]);
        }
        String[] arrays1={"Hello","Everyone","Good morning"};
        for (int i=0;i<arrays.length;i++){
            System.out.println(arrays1[i]);
        }
        // Accessing elements
        System.out.println(arrays[0]);
        // Updating the elements
        arrays[2]=100;
        System.out.println(arrays[2]);
    }
}
